addons/overpressure/CfgEden.hpp

```diff
@@ -0,0 +1,21 @@
+class Cfg3DEN {
+    class Object {
+        class AttributeCategories {
+            class ace_attributes {
+                class Attributes {
+                    class GVAR(overPressureDistanceSetting) {
+                        displayName = CSTRING(distanceCoefficient_displayName);
+                        tooltip = CSTRING(distanceCoefficient_toolTip);
+                        property = QUOTE(overpressureDistanceCoefficient);
+                        control = "Edit";
+                        expression = QUOTE(_this setVariable [ARR_3(QQEGVAR(overpressure,distance),_value,true)]);
```

Comment:
small optimization to not sync var when it's default
```suggestion
                        expression = QUOTE(if (_value >= 0) then { _this setVariable [ARR_3(QQGVAR(distance),_value,true)] });
```

Reply:
@bluefieldcreator Why were these marked as resolved, when they haven't been?

Reply:
@johnb432 My bad, the GitHub phone app is ass